src/mixins/animation.mixin.js

```diff
@@ -73,8 +73,6 @@ fabric.util.object.extend(fabric.StaticCanvas.prototype, /** @lends fabric.Stati
         onComplete();
       }
     });
-
-    return this;
```

Comment:
all the changes to `src/mixins/animation.mixin.js` can't be done because are breaking changes.
We can add them later when we have a major release